project/src/components/comment-form/comment-form.jsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import CommentRating from '../comment-rating/comment-rating.jsx';
+
+function CommentForm() {
+  const [state, setState] = useState({
+    rating: '',
+    review: '',
+  });
+
+  const isValid = !(state.rating.length && state.review.length);
+
+  const onTextChange = (evt) => {
+    const name = evt.currentTarget.name;
+    setState({...state, [name]: evt.target.value});
+  };
```

Comment:
У тебя общий обработчик и на текстареа и на рейтинг, зачем называть onTextChange? Мне кажется onInputChange будет лучше. Значения можно сразу два вытащить, и не совсем понял зачем использовал currentTarget

```suggestion
  const onTextChange = (evt) => {
    const { name, value } = evt.target;
    setState({
      ...state,
      [name]: value
    });
  };
```